### DISCUSSION TITLE
Your "Why" section got a star from me

### DISCUSSION THREAD

**Discussion by adi-g15** [blueedgetechno/win11React#90] [General]
😂

**Comment by blueedgetechno** [comment]
Of couse that's the only thing I focused during the project XD

**Reply by luni-moon** [reply]
That's not the only reason I starred it, but part of it ;)

**Reply by luni-moon** [reply]
Also, the 42 reference :)

**Reply by blueedgetechno** [reply]
The truth, the 42
